combinations/hash.tsv

```diff
@@ -13,8 +13,7 @@ bwa-mem2=2.2.1,samtools=1.12	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.14	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15.1	bgruening/busybox-bash:0.1	0
-hisat2=2.2.0,samtools=1.10	bgruening/busybox-bash:0.1	0
-hisat2=2.2.0,samtools=1.15.1	bgruening/busybox-bash:0.1	0
```

Comment:
We need to keep these lines for old containers here @RaqManzano and just append any new container entries.

Reply:
Will put back those lines then, thanks